cranelift/assembler-x64/meta/src/generate/format.rs

```diff
@@ -0,0 +1,193 @@
+//! Generate format-related Rust code; this also includes generation of encoding
+//! Rust code.
+
+use super::{fmtln, Formatter};
+use crate::dsl;
+
+impl dsl::Format {
+    /// Re-order the Intel-style operand order to accommodate ATT-style
+    /// printing.
+    #[must_use]
+    pub fn generate_att_style_operands(&self) -> String {
```

Comment:
Oh no! I think we all prefer Intel syntax and are kind of stuck with AT&T only because of legacy/inertia. It might be worth listing here or somewhere else why we're doing this for now and what the plan is: e.g. will we remove this swap and update all the compile test expectations once we've migrated all instructions to the new DSL?